packages/tailwindcss-config-analyzer/package.json

```diff
@@ -7,7 +7,7 @@
   "author": "Dani Guardiola",
   "type": "module",
   "dependencies": {
-    "tailwindcss": "^3.4.17"
+    "tailwindcss": "^4.1.3"
```

Comment:
This will break the config analyser. We can merge this for now, but some additional work will be needed later.